test/index.test.js

```diff
@@ -0,0 +1,43 @@
+jest.mock('../ffc-pay-file-consumer/config', () => ({ totalRetries: 1 }))
+const consumer = require('../ffc-pay-file-consumer')
+const mockContext = require('./mock-context')
+jest.mock('../ffc-pay-file-consumer/storage')
+const mockStorage = require('../ffc-pay-file-consumer/storage')
+let message
+const file = {}
```

Comment:
rename to make consistent with `ffc-pay-file-sender` which refers to it as `blob`

also, just remove ambiguity in test names when referring to `blob` or `file`